environments/sbox/sbox.tfvars

```diff
@@ -9,6 +9,11 @@ vnet_rg            = "ss-sbox-network-rg"
 vnet_name          = "ss-sbox-vnet"
 shutter_rg         = "sds-platform-sbox-rg"
 cdn_sku            = "Standard_Verizon"
+ssl_policy = [{
+  policy_type          = "Predefined"
+  policy_name          = "AppGwSslPolicy20170401"
```

Comment:
if we're updating to minimum tls 1.2 - why not use latest policy? https://learn.microsoft.com/en-us/azure/application-gateway/application-gateway-ssl-policy-overview#predefined-tls-policy

Reply:
sure - didn't wanted to jump to the latest directy but I have updated it now :)